packages/wonder-blocks-data/src/__docs__/types.scoped-cache.stories.mdx

```diff
@@ -12,16 +12,103 @@ import {Meta} from "@storybook/addon-docs";
 # ScopedCache
 
 ```ts
-type ScopedCache = {
-    [scope: string]: {
-        [id: string]: ValidCacheData,
-        ...
-    },
-    ...
-};
+interface ScopedCache {
+    set(scope: string, id: string, value: ValidCacheData): void;
 
+    /**
+     * Retrieve a value from the cache.
+     */
+    get(scope: string, id: string): ?ValidCacheData;
+
+    /**
+     * Purge an item from the cache.
+     */
+    purge(scope: string, id: string): void;
+
+    /**
+     * Purge a scope of items that match the given predicate.
+     *
+     * If the predicate is omitted, then all items in the scope are purged.
+     */
+    purgeScope(
+        scope: string,
+        predicate?: (id: string, value: ValidCacheData) => boolean,
+    ): void;
+
+    /**
+     * Purge all items from the cache that match the given predicate.
+     *
+     * If the predicate is omitted, then all items in the cache are purged.
+     */
+    purgeAll(
+        predicate?: (
+            scope: string,
+            id: string,
+            value: ValidCacheData,
+        ) => boolean,
+    ): void;
+}
+```
+
+This interface defines how to interact with a scoped cache, such as [`ScopedInMemoryCache`](/docs/data-exports-scopedinmemorycache--page).
+
+## set()
+
+```ts
+set(
+    scope: string,
+    id: string,
+    value: TValue,
+): void;
+```
+
+Sets a value in the cache within a given scope.
+
+### Throws
+
+| Error Type | Error Name | Reason |
+| ------ | ------ | ------ |
+| [`DataError`](/docs/data-exports-dataerror--page) | `InvalidInputDataError` | `id` and `scope` must be non-empty strings |
+| [`DataError`](/docs/data-exports-dataerror--page) | `InvalidInputDataError` | `value` must be a non-function value |
```

Comment:
**off-topic**: I wish Flow/TS had a way to indicate what errors a function can throw.  Thanks for including it in the docs.  No action req'd.